CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Add a flexible http client providing both contract, and only emulating what's necessary
 - HTTP Client Router: route requests to underlying clients
+- Plugin client and implementations moved here from `php-http/plugins`
```

Comment:
what implementations? (i assume you mean plugin implementations, but that is not clear)
should we say "Plugin client and simple plugins moved here from..."?